### DISCUSSION TITLE
Aggregation Rewrite Attempt - attemptFailed

### DISCUSSION THREAD

**Discussion by ImranDJCS** [DaxStudio/DaxStudio#705] [Q&A]
Hi Darren,
I'm struggling to map one of the columns while configuring Agg table.
DAX Studio is suggesting "attemptFailure" as shown below. I also added how my agg table is configured on two tables.
Could you advise how can over come this unmapped column message?
Thank you

![image](https://user-images.githubusercontent.com/83199486/131969644-8160a4bb-80a5-4bef-9767-b6a7268fa7d1.png)

![image](https://user-images.githubusercontent.com/83199486/131969812-0ee56f72-8950-40dd-9227-517a453729d7.png)

![image](https://user-images.githubusercontent.com/83199486/131969237-d1e05d41-1750-450c-9e3a-c663bdd3fd3a.png)

**Comment by dgosbell** [comment]
It's really hard to say without knowing what the query text was. Were you doing a count() over the NoticeNo column. That is the only mapping I can see in your screenshots so if you were using NoticeNo in some other way that is why the tabular engine would be reporting the rewrite failure
